pages/windows/reg-export.md

```diff
@@ -0,0 +1,11 @@
+# reg export
+
+> Export the specified sub keys, and values into a file.
```

Comment:
Not sure that a comma is needed here.

Reply:
You're right. I think I added that comma due to it previously being a list.